src/kuzya.cpp

```diff
@@ -1361,6 +1375,7 @@ void Kuzya::removeAllNotifications()
 ///***********************************************************************************************************///
 void Kuzya::slotShowNotificationList(bool visible)
 {
+    tabWidget->setVisible(visible);
     notificationList->setVisible(visible);
```

Comment:
I think, we shouldn't hide notificationList inside of widgetTab. If we hide widgetTab, we won't see notificationList, because it is inside of widgetTab. There are no needs to hide notificationList, because we hide widgeTab. NotificationList hides and appears whole all Kuzya. You can find all this by Ctrl+F.